code/modules/mechs/mech_construction.dm

```diff
@@ -1,9 +1,11 @@
 /mob/living/exosuit/proc/dismantle()
 
 	playsound(src.loc, 'sound/items/Deconstruct.ogg', 50, 1)
-	var/obj/structure/heavy_vehicle_frame/frame = new(get_turf(src))
-	for(var/hardpoint in hardpoints)
-		remove_system(hardpoint, force = 1)
+	var/obj/structure/heavy_vehicle_frame/frame = new/obj/structure/heavy_vehicle_frame{
+		is_wired = FRAME_WIRED_ADJUSTED;
```

Comment:
Please don't do it like this. If the is_wired or is_reinforced variable were to be changed, I'm not confident the compiler would catch this.

Reply:
Tell me why compiler doesn't understand his so simple feature as prefabs?

Reply:
This variables is in /obj/structure/heavy_vehicle_frame type, not in structure

Reply:
They isn't setting in /obj/structure/heavy_vehicle_frame/New or Initialize

Reply:
Still not happy with it. Change it

Reply:
I am seconding this.

Reply:
Okey, so